.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!--
+PR checklist:
+- Does this introduce breaking changes?
```

Comment:
hmm 🤔 would be great if the checklist contained actionable items. i don't know if this is actionable. i answer this with yes/no and then what?

Do you mean something like `describe breaking changes in the PR/commit body`?

Reply:
No. This is more a successor to the next question. Probably makes sense to just remove this point and only have the one about the CHANGELOG.md 🤔